### ISSUE TITLE
Incorrect/corrupt tarballs in download_files/

### ISSUE THREAD

**Issue by amarathe84** [Mantevo/mantevo.github.io#2]
The tarballs in download_files/ sub-directory appear incorrect when I git cloned. 

```
$ ls -al
total 104
drwx--S---  2 marathe1 marathe1 4096 Mar 24 12:46 .
drwx--S--- 17 marathe1 marathe1 4096 Mar 24 12:46 ..
-rw-------  1 marathe1 marathe1  131 Mar 24 12:46 CleverLeaf-2.0.tar.gz
-rw-------  1 marathe1 marathe1  132 Mar 24 12:46 CloverLeaf-1.1.tar.gz
-rw-------  1 marathe1 marathe1  132 Mar 24 12:46 CloverLeaf3D-1.0.tar.gz
-rw-------  1 marathe1 marathe1  130 Mar 24 12:46 CloverLeaf3D_ref-1.0.tar.gz
-rw-------  1 marathe1 marathe1  130 Mar 24 12:46 CloverLeaf_ref-1.1.tar.gz
-rw-------  1 marathe1 marathe1  131 Mar 24 12:46 CoMD_Ref.tgz
-rw-------  1 marathe1 marathe1  129 Mar 24 12:46 EpetraBenchmarkTest-1.0.tar.gz
-rw-------  1 marathe1 marathe1  130 Mar 24 12:46 HPCCG-1.0.tar.gz
-rw-------  1 marathe1 marathe1  131 Mar 24 12:46 MiniAero-v1.0-Mantevo.tar.gz
-rw-------  1 marathe1 marathe1  133 Mar 24 12:46 PathFinder_1.0.0.tgz
-rw-------  1 marathe1 marathe1  131 Mar 24 12:46 TeaLeaf-1.0.tar.gz
-rw-------  1 marathe1 marathe1  130 Mar 24 12:46 TeaLeaf_ref-1.0.tar.gz
-rw-------  1 marathe1 marathe1  134 Mar 24 12:46 Test_case_5kx5k.tar.gz
-rw-------  1 marathe1 marathe1  134 Mar 24 12:46 Test_case_7kx7k.tar.gz
-rw-------  1 marathe1 marathe1  130 Mar 24 12:46 miniAMR_1.0_all.tgz
-rw-------  1 marathe1 marathe1  130 Mar 24 12:46 miniAMR_1.0_ref.tgz
-rw-------  1 marathe1 marathe1  132 Mar 24 12:46 miniFE-2.0.1.tgz
-rw-------  1 marathe1 marathe1  131 Mar 24 12:46 miniFE_ref-2.0.tgz
-rw-------  1 marathe1 marathe1  132 Mar 24 12:46 miniGhost_1.0.1.tar.gz
-rw-------  1 marathe1 marathe1  132 Mar 24 12:46 miniGhost_ref_1.0.1.tar.gz
-rw-------  1 marathe1 marathe1  131 Mar 24 12:46 miniMD_1.2.tgz
-rw-------  1 marathe1 marathe1  131 Mar 24 12:46 miniMD_ref-2.0.tgz
-rw-------  1 marathe1 marathe1  131 Mar 24 12:46 miniSMAC2D-2.0.tgz
-rw-------  1 marathe1 marathe1  130 Mar 24 12:46 miniXyce_1.0.tar.gz
```

Each of them has a 3-line description of the file (SHA256, size) and a URL of Git LFS. Have the tarballs been moved somewhere else? This affects spack/spack#15291.

**Comment by junghans** [comment]
@jwillenbring could you please have a look? We need an easy to download these tarballs in Spack for ECP Proxy.

**Comment by amarathe84** [comment]
Hi @jwillenbring: Was there any update on this issue? The Spack builds for Mantevo benchmarks are still failing (spack/spack#15291). Thanks!

**Comment by jwillenbring** [comment]
The old server no longer exists. The Mantevo site was migrated to github.io. I did take a backup of the old site before we migrated in October of last year. We will need to find a new place to host the tarballs. There is a new server that replaced the old one. I will have to check if any mantevo content ended up on that server or not.

**Comment by junghans** [comment]
The tarballs aren't that big, just check them into git. (I know, the horror!!!!!)

**Comment by amarathe84** [comment]
@jwillenbring Thanks for the update! If the tarballs are on a public server somewhere or you end up checking in the barballs back into this repository (as @junghans mentioned), I'll be happy to update the Spack packages to point to the new links.

**Comment by junghans** [comment]
@jwillenbring ping

**Comment by maherou** [comment]
@junghans, If you see a path to getting these tarfiles built and placed, please feel free to submit a pull request.  I am guessing these requests are getting buried in Jim’s inbox.  If you don’t have enough details to make it happen, let me know what you need done.  I might be able to help.

**Comment by junghans** [comment]
@maherou, ok, I was trying to get the files from git-lfs and check them in, but it seems you guys are over the GitHub data quota:
```
git clone git@github.com:Mantevo/mantevo.github.io.git
Cloning into 'mantevo.github.io'...
remote: Enumerating objects: 358, done.
remote: Total 358 (delta 0), reused 0 (delta 0), pack-reused 358
Receiving objects: 100% (358/358), 5.40 MiB | 6.07 MiB/s, done.
Resolving deltas: 100% (135/135), done.
Downloading download_files/CleverLeaf-2.0.tar.gz (414 KB)
Error downloading object: download_files/CleverLeaf-2.0.tar.gz (0af52fe): Smudge error: Error downloading download_files/CleverLeaf-2.0.tar.gz (0af52fe3d5b5a783eb9fa086b135fbf208bb6a6b0daf55922d51725e2795d4ac): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
```

**Comment by maherou** [comment]
Is the data quota the real problem, or are the tarballs truly corrupt?  Just making sure.

Some time ago I saw the warnings about approaching the quota, but I did not and still don’t know what ate the Mantevo data budget.

On May 17, 2020, at 7:57 AM, Christoph Junghans <notifications@github.com> wrote:

﻿

@maherou<https://github.com/maherou>, ok, I was trying to get the files from git-lfs and check them in, but it seems you guys are over the GitHub data quota:

git clone git@github.com:Mantevo/mantevo.github.io.git
Cloning into 'mantevo.github.io'...
remote: Enumerating objects: 358, done.
remote: Total 358 (delta 0), reused 0 (delta 0), pack-reused 358
Receiving objects: 100% (358/358), 5.40 MiB | 6.07 MiB/s, done.
Resolving deltas: 100% (135/135), done.
Downloading download_files/CleverLeaf-2.0.tar.gz (414 KB)
Error downloading object: download_files/CleverLeaf-2.0.tar.gz (0af52fe): Smudge error: Error downloading download_files/CleverLeaf-2.0.tar.gz (0af52fe3d5b5a783eb9fa086b135fbf208bb6a6b0daf55922d51725e2795d4ac): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

**Comment by jwillenbring** [comment]
The issue is the quota. We are way over. This actually has nothing to do then with moving the .org server that mantevo.org was formerly hosted on. I am looking at hosting the files elsewhere. Git LFS seems to not support the bandwidth we apparently need. One thing that would help our bandwidth a lot would be to use a Spack binary cache. As these tarballs don't change, depending on how things are set up in Spack, it may not be necessary to pull them from the server everyday.

**Comment by maherou** [comment]
Can we put these files on Zenodo?  It is a persistent large-scale data storage service.

**Comment by junghans** [comment]
I would just commit them to plain old git without lfs as we did [here](https://github.com/votca/downloads), this doesn't count against the quota. 

Then we just change the url in spack to use the GitHub's `raw` link.

GitHub has a 40MB (?) blob limit, so you will need to commit one tarball at the time and even then  we had success with raising the limit to 100MB (for lammps) in the past.

**Comment by jwillenbring** [comment]
It seems so odd to put files back in the repo because the large file service doesn't support the bandwidth issues as well as just checking them in. My understanding is that this is done by file extension, so it is not possible to move just some files away from LSF. In any case, I am getting some admin support for a different place to host the files and will follow up once that is worked out.

**Comment by jwillenbring** [comment]
I moved the tarballs to the new server. I also updated the download pages on the mantevo.github.io site to point to the new download location. These pages were not using the Git LFS links. I have not touched anything regarding the LFS files in the mantevo.github.io repo. 

The new links are of the form

https://downloads.mantevo.org

instead of the old ones, which were of the form

https://mantevo.org/downloads

the rest of all the URLS remains the same. Note that I did not change the practice we used formerly to count downloads, that is point downloads to an html file that redirects to the tarball itself, instead of just linking to the tarball. With the LFS files, I am guessing the counts are no longer accurate, but I did not make any modifications to this approach.

So there are a few options

1. point to the files at downloads.mantevo.org
1. check the files into the repo directly, instead of using the LFS links
1. set up a new repo in the mantevo organization solely for storing the files.
